docs/LogGuide.md

```diff
@@ -557,7 +559,8 @@ ACT Log Line Examples:
 
 This combatant may be invisible and fake.  The real ones have more HP.
 For example, at the start of Deltascape V2.0 you will see messages like the
-latter 5 examples above.
+latter 5 examples above. In addition, anything with an NPC ID of 9020 (0x233C)
+is almost certainly fake.
```

Comment:
Do you have examples of this? I haven't seen anything specific related to fake NPC ids before.

Reply:
https://discord.com/channels/192732719879290880/192732719879290880/926929235766030346 (fflogs discord)

Reply:
The majority of BNPC Base IDs aren't used by "real" actors, but you also can't rely on the ID - battle NPC actors are identified by the combination of their Base+Model IDs. While 9020 is likely one of these cases, so are like 80% of the other IDs. You can't actually rule them out without the model ID as well though.